### DISCUSSION TITLE
Syncing Multiple Plays

### DISCUSSION THREAD

**Discussion by teufelhunden0311** [Taxel/PlexTraktSync#2095] [Q&A]
Hello,

I have a question about this. I have this setup with the Unraid community app and I know that is unsupported but I have it working and it will sync a previously unwatched show to Trakt but if I were to rewatch an episiode of something that was already marked as watched at a previous point in time on Trakt it doesn't sync the new viewing. 

I read the documentation and either couldn't find it or didn't understand it when I did come across it so is this even possible?  Can it record multiple viewings of the same movie or show?

**Comment by macky06** [comment]
I have the same concern, if someone can told us how can we have the multiple viewing please

**Comment by glensc** [comment]
this has been asked and answered several times:
1. plextraktsync sync - can't do rewatches
1. plextraktsync watch - records rewatches

ps: from readme:
> you can also just [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to improve documentation
